rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2023-08-24" # rustc 1.72.0 (2023-08-24)
+channel = "nightly-2023-05-22"  # rustc 1.71.0 8b4b20836
 components = [ "rustfmt", "clippy" ]
 targets = [ "wasm32-unknown-unknown" ]
 profile = "minimal"
```

Comment:
Just wondering if being a library repo we really need this. I think not

@wischli

Reply:
You are probably right. I followed the path of Substrate which is not necessarily working with latest nightly versions. Feel free to remove the toolchain and adjust CI.